webapp/pages/post/_id/_slug/index.vue

```diff
@@ -90,7 +90,17 @@
           @toggleNewCommentForm="toggleNewCommentForm"
         />
         <ds-space margin-bottom="large" />
-        <hc-comment-form v-if="showNewCommentForm" :post="post" @createComment="createComment" />
+        <comment-form
+          v-if="showNewCommentForm && !post.author.blocked"
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-29T16:06:30Z" title="Wednesday, January 29th 2020, 5:06:30 pm +01:00">Jan 29, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -90,7 +90,17 @@
           @toggleNewCommentForm="toggleNewCommentForm"
         />
         <ds-space margin-bottom="large" />
-        <hc-comment-form v-if="showNewCommentForm" :post="post" @createComment="createComment" />
+        <comment-form
+          v-if="showNewCommentForm && !post.author.isBlocked"
```


I'm just looking at the frontend right now, so not sure where this is coming from – but this reads like "when I have blocked the post author" when it should be "when the post author has blocked me". Or am I wrong?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-29T20:06:46Z" title="Wednesday, January 29th 2020, 9:06:46 pm +01:00">Jan 29, 2020</time>_

---


it is reciprocal @alina-beck ... we could think about renaming it to make it clear... 
maybe `blockedExists` or `hasBlocked`
`post.authord.hasBlocked` works well, but `user.hasBlocked`, maybe not so much??

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-30T09:32:14Z" title="Thursday, January 30th 2020, 10:32:14 am +01:00">Jan 30, 2020</time>_

---


I just renamed it to `blocked`, which could stand for `isOrWasBlocked` ... didn't really love any of the names I could come up with to describe it..